docs/end2end-TrecQA-eval.md

```diff
@@ -33,16 +32,22 @@ Path of the index
 
 
 ```
--hits (optional: positive integer)
+-h0 (optional: positive integer)
+```
+
+Number of documents to be retrieved per question (default: 1000)
+
+```
+-h1 (optional: positive integer)
 ```
 
-Number of passages to be retrieved per question (default: 20)
+Top-h1 passages to be re-ranked by the answer extractor (default: 10)
 
 ```
--k (optional: positive integer)
+-k (optional: list of positive integers)
 ```
 
-Top-k passages presented by the system (default: 10)
+Evaluate at depth k (default: 10)
```

Comment:
Sorry for the terminology, it's supposed to mean create run files that have at most 5 answers. I'll have this updated.